pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -203,6 +210,17 @@ func (r *reportingPlugin) Outcome(outctx ocr3types.OutcomeContext, query types.Q
 		}
 	}
 
+	// Since we will most likely get N different timestamps, each with frequency=1, we get the median instead of the mode.
+	sortkeys.Uint32s(sortedTimestamps)
```

Comment:
Is this doing anything special? I think that with generics we could just use the std library, since utint32 is `cmp.Ordered`:
```suggestion
	slices.Sort(sortedTimestamps)
```